### PR TITLE
Using the Tracks client from Share, track Today Widget usage.

### LINKED ISSUE #3306: Track Today Extension usage in analytics
Sounds easier than it is since we don't include the analytics pod and it sort of is a separate app.

### PR DESCRIPTION
**Pull request by astralbodies**
Closes #3306 

To test:
1. Launch the today widget.
2. When unconfigured, tap configure. Verify event sent.
3. When configured, verify event sent when data is refreshed.
4. When configured, verify event sent when tapping data to launch containing app.

Needs review: @jleandroperez

### CODE REVIEW

File: WordPress/WordPressTodayWidget/Tracks+TodayWidget.swift [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+    private func trackExtensionEvent(event: ExtensionEvents, properties: [String: AnyObject]? = nil) {
+        track(event.rawValue, properties: properties)
+    }
+    
+    
+    // MARK: - Private Enums
+    private enum ExtensionEvents : String {
+        case Accessed          = "wpios_today_extension_accessed"
+        case StatsLaunched     = "wpios_today_extension_stats_launched"
+        case ConfigureLaunched = "wpios_share_extension_configure_launched"
```
Comment (jleandroperez) [inline WordPress/WordPressTodayWidget/Tracks+TodayWidget.swift]: Should this one read `wpios_share_extension` ooor `wpios_today_extension` ?

**Comment by jleandroperez** [comment]
Events showing up in Tracks. All looking good sir!

:shipit:

**Comment by astralbodies** [comment]
Thank you kindly, @jleandroperez! Appreciate the review.

**Comment by jleandroperez** [comment]
Anytime @astralbodies!

**Pull request merged**